GoogleForCreators/web-stories-wp
# Floating Menu: Add opacity input

## Summary

Add opacity input to floating menu

## Relevant Technical Choices

Original opacity input allows negative numbers. This input will also allow negative numbers to match that functionality, but in the future I think we should limit both to 0. Better done in another ticket.

## To-do

n/a

## User-facing changes

![opacity](https://user-images.githubusercontent.com/22185279/157753599-352bf8d6-b572-4d59-a0da-38e774e546b5.gif)

## Testing Instructions






This PR can be tested by following these steps:

1. Enable floating menu experiment
2. Add an element to the canvas
3. Click on the element. Should see the opacity input
4. Type a number in and tab away. Should update if it's between -infinity and 100
5. Type a letter in and tab away. Should revert to previous value
6. Select multiple elements. Should not see the input in the floating menu.

## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #10633
